modules/ROOT/pages/dw-core-annotations.adoc

```diff
@@ -1,11 +1,10 @@
 = Core Annotations (dw::Core)
 
-[%header,cols="2,2,4a"]
 |===
 | Annotation | Definition | Description
 
 | AnnotationTarget
-| `@AnnotationTarget(targets: Array<String>)`
+| `@AnnotationTarget(targets: Array<"Function" | "Parameter" | "Variable" | "Import">)`
```

Comment:
TODO: copyedit and see if it's possible to add an intro somewhere in the source code.